src/enums/amalgamationEnums.ts

```diff
@@ -2,12 +2,16 @@ export enum AmlStatuses {
   OK,
   ERROR_CCC_MISMATCH,
   ERROR_FOREIGN,
+  ERROR_FOREIGN_HORIZONTAL,
   ERROR_FOREIGN_UNLIMITED,
+  ERROR_FOREIGN_UNLIMITED2,
+  ERROR_FOREIGN_UNLIMITED3,
```

Comment:
I'd like to rename these but I think these rules will change anyways (I think they may be reduced to just 1 rule). Leaving this for now.